includes/admin/debug-tools/class-wcs-notifications-debug-tool-processor.php

```diff
@@ -210,7 +214,7 @@ public function get_next_batch_to_process( int $size ): array {
 	 * @param array $batch Batch to process, as returned by 'get_next_batch_to_process'.
 	 */
 	public function process_batch( array $batch ): void {
-		// This is a bit unnecessary. Perhaps convert `update_status` to static to avoid instantiating the class?
+
 		$subscriptions_notifications = new WCS_Action_Scheduler_Customer_Notifications();
 
```

Comment:
Done https://github.com/Automattic/woocommerce-subscriptions-core/pull/689/commits/2332026e77506e4a944ae51dc3d92102704e6f18

Reply:
@peterfabian I might be missing something, but could we avoid using a processor altogether and instead just remove every AS action for a given group via `as_unschedule_all_actions()`. Essentially, calling the `ActionScheduler_Store::instance()->cancel_actions_by_group()` behind the scenes? 

Same for the batch processor obviously.

Reply:
No, I don't think you're missing anything. You're right that fewer abstractions and function calls are better. We can just call `as_unschedule_all_actions( WCS_Action_Scheduler_Customer_Notifications::$notifications_as_group )`. 

Essentially I just used it to encapsulate the group so that it's not a public property, but we can probably just make it public and call the AS function directly.